README.md

```diff
@@ -128,12 +128,15 @@ $ omf install https://github.com/jhillyerd/plugin-git
 | glr          | `git pull --rebase`                                  |
 | gp           | `git push`                                           |
 | gp!          | `git push --force-with-lease`                        |
+| gpo          | `git push origin`                                    |
 | gpv          | `git push --no-verify`                               |
 | gpv!         | `git push --no-verify --force-with-lease`            |
 | ggp          | push origin _current-branch_                         |
+| ggf          | `ggp --force`                                        |
+| ggfl         | `ggp --force-with-lease`                             |
```

Comment:
We've semi standardized on "force-with-lease", and using a `!` to indicate that.  So far I haven't had any problems using lease since we switched to it, never needed an full force.  I think if people really want to force it without a lease, they should type out `--force`.

Suggest we drop `ggf`, and rename `ggfl` to `gpo!` - what do you think?

Reply:
I'm good with it :D.

Reply:
Since `ggfl` is aliased to `ggp --force-with-lease`. And `gpo` is just `git push origin`, I think `ggp!` is more suitable - and also follow the convention.

I'm good with dropping `ggf`.

Reply:
Implemented here: https://github.com/jhillyerd/plugin-git/pull/39/commits/05a12235bfee675f8216b58f029af7e462e1dc0a